jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/DefaultGenerationConfig.java

```diff
@@ -117,6 +117,11 @@ public AnnotationStyle getAnnotationStyle() {
     }
 
     /**
+     * @return {@link InclusionLevel#NON_NULL}
+     */
+    public InclusionLevel getInclusionLevel() { return InclusionLevel.NON_NULL; }
```

Comment:
This project doesn't use single line blocks like this, could you change the style?